packages/backend/test/peripherals/database/setup.ts

```diff
@@ -6,12 +8,27 @@ export function setupDatabaseTestSuite() {
   const config = getConfig('test')
   const knex = DatabaseService.createKnexInstance(config.databaseUrl)
   const skip = config.databaseUrl === __SKIP_DB_TESTS__
+  let schemaName = ''
 
   before(async function () {
     if (skip) {
       this.skip()
     } else {
-      await knex.migrate.latest()
+      // For describe("one", () => describe("two") => {}) test suite we set up
+      // 'test_one_two' schema before running tests.
+
+      const titlePath = this.test?.parent?.titlePath()
+      schemaName = `test_${titlePath?.join('_') || uuid()}`
+
+      log('Creating test schema', schemaName)
+
+      // We drop before instead of after tests, so we can inspect the
+      // contents of database when tests fail.
+      await knex.raw(`DROP SCHEMA IF EXISTS "${schemaName}" CASCADE`)
```

Comment:
You should use knex raw's parameters not string interpolation. This way no matter what the describe text is the query won't break

Reply:
I changed `DROP` and `CREATE` to use Knex parameter binding, but I'm unable to write `SET SCHEMA` like this :/